tst/EnergyPlus/unit/ReportSizingManager.unit.cc

```diff
@@ -169,27 +173,85 @@ TEST_F(EnergyPlusFixture, ReportSizingManager_GetCoilDesFlowT)
     DataSizing::SysSizInput(1).CoolCapControl = DataSizing::Bypass;
     DataSizing::CalcSysSizing(1).CoolZoneAvgTempSeq(1) = 13;
     DataSizing::CalcSysSizing(1).MixTempAtCoolPeak = 15;
     ReportSizingManager::GetCoilDesFlowT(sysNum, CpAir, designFlowValue, designExitTemp);
+    EXPECT_FALSE(has_err_output(true));
     EXPECT_DOUBLE_EQ(10, designExitTemp);
     EXPECT_NEAR(0.119823, designFlowValue, 0.0001);
 
-    // tear down
-    DataSizing::DataAirFlowUsedForSizing = 0.0;
-    DataSizing::CalcSysSizing(1).SumZoneCoolLoadSeq.deallocate();
-    DataSizing::CalcSysSizing(1).CoolZoneAvgTempSeq.deallocate();
-    DataSizing::SysSizPeakDDNum(1).TimeStepAtSensCoolPk.deallocate();
-    DataSizing::SysSizPeakDDNum(1).TimeStepAtCoolFlowPk.deallocate();
-    DataSizing::SysSizPeakDDNum(1).TimeStepAtTotCoolPk.deallocate();
-    DataSizing::SysSizInput.deallocate();
-    DataSizing::SysSizPeakDDNum.deallocate();
-    DataSizing::FinalSysSizing.deallocate();
-    DataSizing::CalcSysSizing.deallocate();
+}
+TEST_F(EnergyPlusFixture, ReportSizingManager_GetCoilDesFlowT_NoPeak)
+{
+    // setup global allocation
+    DataSizing::SysSizInput.allocate(1);
+    DataSizing::SysSizPeakDDNum.allocate(1);
+    DataSizing::FinalSysSizing.allocate(1);
+    DataSizing::CalcSysSizing.allocate(1);
+    DataSizing::CalcSysSizing(1).SumZoneCoolLoadSeq.allocate(1);
+    DataSizing::CalcSysSizing(1).CoolZoneAvgTempSeq.allocate(1);
+    DataSizing::SysSizPeakDDNum(1).TimeStepAtSensCoolPk.allocate(1);
+    DataSizing::SysSizPeakDDNum(1).TimeStepAtCoolFlowPk.allocate(1);
+    DataSizing::SysSizPeakDDNum(1).TimeStepAtTotCoolPk.allocate(1);
+
+    // one-time global initialization
+    int const DesignDayForPeak = 0;
+    DataSizing::SysSizPeakDDNum(1).SensCoolPeakDD = DesignDayForPeak;
+    DataSizing::SysSizPeakDDNum(1).CoolFlowPeakDD = DesignDayForPeak;
+    DataSizing::SysSizPeakDDNum(1).TotCoolPeakDD = DesignDayForPeak;
+    DataSizing::FinalSysSizing(1).CoolSupTemp = 10;
+    DataSizing::FinalSysSizing(1).MassFlowAtCoolPeak = 2.0;
+    DataSizing::FinalSysSizing(1).DesCoolVolFlow = 0.15;
+    DataSizing::DataAirFlowUsedForSizing = 0.2;
+    DataEnvironment::StdRhoAir = 1000;
+    DataSizing::CalcSysSizing(1).SumZoneCoolLoadSeq(1) = 1250000;
+
+    // one-time argument initialization
+    int const sysNum = 1;
+    Real64 const CpAir = 4179;
+
+    // argument return values
+    Real64 designFlowValue;
+    Real64 designExitTemp;
+
+    DataSizing::SysSizInput(1).CoolingPeakLoadType = DataSizing::TotalCoolingLoad;
+    DataSizing::FinalSysSizing(1).CoolingPeakLoadType = DataSizing::TotalCoolingLoad;
+
+    // Single path for VAV
+    DataSizing::SysSizInput(1).CoolCapControl = DataSizing::VAV;
+    ReportSizingManager::GetCoilDesFlowT(sysNum, CpAir, designFlowValue, designExitTemp);
+    EXPECT_FALSE(has_err_output(true));
+    EXPECT_DOUBLE_EQ(DataSizing::FinalSysSizing(1).CoolSupTemp, designExitTemp);
+    EXPECT_DOUBLE_EQ(0.002, designFlowValue);
```

Comment:
Must be some 0's in the data used to calculate designFlowValue = 0.002, but it passes.

Reply:
Umm, this shouldn't be in here

Reply:
Oh wait - never mind - all good here. Forgot what was in here.